### ISSUE TITLE
NPM peer dependencies warning

### ISSUE THREAD

**Issue by jimblue** [elisherer/webpack-dev-server-waitpage#19]
Hello,

I have `"webpack": "4.44.1"` as `devDependencies` in my `package.json`

But I get the following warning:

```bash
npm WARN webpack-dev-server-waitpage@2.0.0 requires a peer of webpack@^4 | ^5 but none is installed. You must install peer dependencies yourself.
```

Do you know why, is it because of `webpack-dev-server-waitpage` configuration ?

Thank

**Comment by elisherer** [comment]
I think I got the "OR" operator wrong. I published 2.0.1 with `||` instead of `|` can you please try and tell if it's resolved?

**Comment by jimblue** [comment]
yep good job it's fixed !
